src/routes/repetition/components/RepetitionProgressChart.tsx

```diff
@@ -0,0 +1,71 @@
+import { useFocusEffect } from '@react-navigation/native'
+import React, { ReactElement, useCallback } from 'react'
+import { heightPercentageToDP as hp } from 'react-native-responsive-screen'
+import { LineSegment, VictoryAxis, VictoryBar, VictoryChart, VictoryLabel } from 'victory-native'
+
+import theme from '../../../constants/theme'
+import useLoadAsync from '../../../hooks/useLoadAsync'
+import { getNumberOfWordsInEachSection, sections } from '../../../services/RepetitionService'
+import { getLabels } from '../../../services/helpers'
+
+const { chartColor1, chartColor2, chartColor3, chartColor4 } = theme.colors
+const barColors = [chartColor1, chartColor2, chartColor2, chartColor2, chartColor2, chartColor2, chartColor3]
+const RepetitionProgressChart: () => ReactElement = () => {
```

Comment:
is there a reason not to use the standard react fc annotation?
```suggestion
const RepetitionProgressChart = (): ReactElement => {
```